express/src/services/lambda-facade/StubLambdaFacade.ts

```diff
@@ -22,12 +23,18 @@ class StubLambdaFacade implements LambdaFacadeInterface {
         phone: {S: "07700 987 654"}
     };
 
-    private readonly defaultPublicKey =
+    private DEFAULT_PUBLIC_KEY =
```

Comment:
Why this change? The default key never changes so it should be `readonly`

Reply:
Not sure how this happened, looks like blip in git merges as I don't remember either of us actually changing that; the readonly version should stand.

Reply:
@CharlesIC , I have reverted the change, all tests pass fine .